### DISCUSSION TITLE
Issue with Premium Subscription Access via Docker Login

### DISCUSSION THREAD

**Discussion by kevinnadar22** [devlikeapro/waha#520] [General]
I'm experiencing an issue with my premium subscription on Waha. The Docker login credentials provided are no longer working, preventing me from accessing the service.

Could you please verify or update my login details? Your prompt assistance would be greatly appreciated.

```
Error: WARNING! Using --password via the CLI is insecure. Use --password-stdin.
Error response from daemon: Get "https://registry-1.docker.io/v2/": unauthorized: incorrect username or password
```

**Username : devlikeapro
Password: dckr_pat_**

**Comment by devlikepro** [comment]
Hi! I can not find any records for jesikamaraj@gmail.com.
Could you send us your email? waha@devlike.pro

Keep in mind that in order to make `docker login` work  you need to be an active subscriber on Boosty/Patreon or using Crypto

[FAQ: is it one time purchase?](https://waha.devlike.pro/docs/overview/faq/#i-want-to-purchase-waha-plus-waha-pro-is-it-one-time-purchase)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
